ex2.js

```diff
@@ -0,0 +1,24 @@
+const printSum = function (arr) {
+  let suma = 0;
+  for (let i = 0; i < arr.length; i ++) {
+    suma += arr[i];
+  }
+  return suma;
+}
+
+const printSumDoi = function (arr) {
+  let suma = 0;
+  arr.map((elem) => {
```

Comment:
```suggestion
  arr.each((elem) => {
```

desi functioneaza si asa , functia map are alta abordare: si anume: ea returneaza ceva